addons/binding/org.openhab.binding.evohome/src/main/java/org/openhab/binding/evohome/discovery/EvohomeDiscoveryService.java

```diff
@@ -54,18 +59,16 @@ public void startScan() {
         if (evohomeBridgeHandler != null) {
             try {
                 EvohomeApiClient client = evohomeBridgeHandler.getApiClient();
-                // TODO Maybe client.update()
                 if (client != null) {
-
+//                    client.update();
```

Comment:
That that can happen but I was finding that even though the scan is run manually after startup it was returning null data. Perhaps the refresh isn't happening properly?